skaffold.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: skaffold/v4beta6
+kind: Config
+metadata:
+  name: che-dashboard
+build:
+  tagPolicy:
+    sha256: {}
+  artifacts:
+    - image: che-dashboard
+      context: .
+      custom:
+        buildCommand: |
+          podman build  --tag=$IMAGE -f build/dockerfiles/skaffold.Dockerfile . && podman push $IMAGE
```

Comment:
Could you prepare a script, like this one ```che-dashboard/scripts
/dash-licenses.sh```, which helps to choose automatically what to use **Docker** or **Podman**?

Reply:
done